modules/Core/pages/panel/registration.php

```diff
@@ -163,7 +163,8 @@
     $oauth_provider_data[$provider_name] = [
         'enabled' => NamelessOAuth::getInstance()->isEnabled($provider_name),
         'setup' => NamelessOAuth::getInstance()->isSetup($provider_name),
-        'icon' => $provider_data['icon'],
+        'icon' => $provider_data['icon'] ?? null,
```

Comment:
Forgive me here but is `?? null` not necessary because `??` will only evaluate as true if `$provider_data['icon']` is already null?